DQMServices/FileIO/python/DQMFileSaverPB_cfi.py

```diff
@@ -1,7 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
 # DQM file saver module
-dqmSaver = cms.EDAnalyzer("DQMFileSaverPB",
+
+# Since we are disabling the AnelasticDQM process in the DQM online machines
+# we are going to start accumulating PB files there.
+# The simplest solution to avoid accumulating files is to disable the PB
+# file production in all clients, since the new GUI is not deployes anyway, 
+# by replacing the dqmSaver with a stub dqmSaver.
+# dqmSaver = cms.EDAnalyzer("DQMFileSaverPB",
+
+dqmSaver = cms.EDAnalyzer("DQMFileSaverPBStub",
```

Comment:
This would remove the original ```DQMFileSaverPB```  from being seen by ConfDB parsing (no cfi file with it!) and thus no longer available for HLT menus online or offline. As Marco mentioned we want to use the plugin for online, so we'd need another solution here, for example, a new separate cfi file ```DQMFileSaverPBStup_cfi.py```, and not removing the old one.